guardrails/utils/pydantic_utils.py

```diff
@@ -2,12 +2,8 @@
 
 Guardrails lets users specify
 
-<pydantic
-    model="Person"
-    name="person"
-    description="Information about a person."
-    on-fail-pydantic="reask" / "refrain" / "raise"
-/>
+<pydantic     model="Person"     name="person" description="Information
```

Comment:
Can you undo this? I'm guessing this change was added by the linter, but it makes the docstring less readable.

Reply:
Done!